README.md

```diff
@@ -172,6 +173,7 @@ Contributions are welcome!
 - [Forth](https://wiki.forth-ev.de/doku.php/en:projects:microbit:start) - Forth, a stack-based language, for the BBC micro:bit.
 - [Pascal](http://wiki.freepascal.org/micro:bit) - Free Pascal compiler that can target the ARM embedded platform, including the micro:bit.
 - [Ada](https://github.com/AdaCore/Ada_Drivers_Library/tree/master/examples/MicroBit) - Instruction on how to setup the Ada development environment for the micro:bit.
+- [Sniff](http://www.sniff.org.uk) - Sniff is a "Scratch-like" programming language that's designed to help Scratchers move gently from Scratch to more conventional languages.
```

Comment:
It's not clear from that link how to use Sniff with the micro:bit, is there any article or section in the website that clarifies this?

Reply:
Closest to that is this page: http://www.sniff.org.uk/p/bbc-microbit.html
It should be better described because it targets Scratchers but that's what author gave us. Should I change link to that subpage?

Reply:
Yes, it looks like that might be the best link for info on compiling for the micro:bit. If can change the link to that I'll merge the PR :)

Reply:
Done. :)
P.S. Thanks for this nice list od resources.